src/sentry/web/frontend/twofactor.py

```diff
@@ -18,6 +18,10 @@
 class TwoFactorAuthView(BaseView):
     auth_required = False
 
+    def _get_org_from_user(self, user):
```

Comment:
since you're returning multiple orgs, i would rename this `_get_orgs_from_user`.

better yet, you can use the `User` model's `get_orgs` function so you don't need this function. Always good to check the model for useful helper functions!